ksqldb-api-client/src/main/java/io/confluent/ksql/api/client/impl/DdlDmlRequestValidators.java

```diff
@@ -34,7 +37,10 @@ static boolean validateExecuteStatementRequest(
       return false;
     }
 
-    if (sql.indexOf(";") != sql.lastIndexOf(";")) {
+    if (Arrays.stream(sql.split(QUOTED_STRING_OR_SEMICOLON))
```

Comment:
nit: improve readability by pulling this logic out into a method called `containsMultipleStatements(sql)` (or similar).